src/screens/SendToken/SendTokenTransaction.js

```diff
@@ -166,87 +170,91 @@ class SendTokenTransaction extends React.Component<Props> {
     const successButtonText = transactionType === TRANSACTION_TYPE.EXCHANGE
       ? t('button.finish')
       : t('button.magic', { exclamation: true });
     return (
-      <ButtonWrapper>
-        <Button onPress={this.handleDismissal} title={successButtonText} />
-      </ButtonWrapper>
+      <ButtonContainer>
+        <ButtonWrapper>
+          <Button onPress={handleDismissal} title={successButtonText} />
+        </ButtonWrapper>
+        <TouchableOpacity onPress={handleDismissal}>
+          <Button variant="text" title={t('button.viewOnBlockchain')} onPress={viewOnBlockchain} />
+        </TouchableOpacity>
+      </ButtonContainer>
     );
-  }
+  };
 
-  renderFailureButtons = () => {
-    const { noRetry } = this.props.navigation.state.params;
+  const renderFailure = () => {
     return (
-      <FailureButtonsWrapper>
+      <ButtonContainer>
         {!noRetry && (
           <ButtonWrapper>
-            <Button onPress={this.handleNavigationBack} title={t('button.retry')} />
+            <Button onPress={handleNavigationBack} title={t('button.retry')} />
           </ButtonWrapper>
         )}
-        <TouchableOpacity onPress={this.handleDismissal}>
+        <TouchableOpacity onPress={handleDismissal}>
           <CancelText>{t('button.cancel')}</CancelText>
         </TouchableOpacity>
-      </FailureButtonsWrapper>
+      </ButtonContainer>
     );
-  }
+  };
 
-  render() {
-    const { navigation } = this.props;
-    const {
-      isSuccess,
-      error,
-      transactionPayload: {
-        tokenType: transactionTokenType,
-        extra: {
-          allowance = {},
-        } = {},
-      },
-      transactionType,
-    } = navigation.state.params;
-
-    const animationSource = isSuccess ? animationSuccess : animationFailure;
-    const transactionStatusText = isSuccess
-      ? getTransactionSuccessMessage(transactionType)
-      : getTransactionErrorMessage(error);
-    const isAllowanceTransaction = Object.keys(allowance).length;
-    const transactionStatusTitle = isSuccess
-      ? getTransactionSuccessTitle({ transactionTokenType, transactionType, isAllowanceTransaction })
-      : t('error.transactionFailed.default');
-    const titleStyle = { ...objectFontStyles.large, marginTop: 16, marginBottom: 7 };
-    const textStyle = { ...objectFontStyles.regular, marginBottom: 75 };
-    return (
-      <Container>
-        <Wrapper flex={1} center regularPadding>
-          <Animation source={animationSource} />
-          <Title fullWidth title={transactionStatusTitle} align="center" noBlueDot titleStyles={titleStyle} noMargin />
-          <Paragraph light center style={textStyle}>{transactionStatusText}</Paragraph>
-          {isSuccess ? this.renderSuccessButton() : this.renderFailureButtons()}
-        </Wrapper>
-        {/*
+  const animationSource = isSuccess ? animationSuccess : animationFailure;
+  const transactionStatusText = isSuccess
+    ? getTransactionSuccessMessage(transactionType, chainType)
```

Comment:
You need to get user-friendly name for chain to properly format user message.
```suggestion
    ? getTransactionSuccessMessage(transactionType, chainConfig.titleShort)
```